django/applications/catmaid/management/commands/catmaid_export_data.py

```diff
@@ -80,10 +81,10 @@ def collect_data(self):
         if not check_tracing_setup(self.project.id, classes, relations):
             raise CommandError("Project with ID %s is no tracing project." % self.project.id)
 
-        exclude_skeleton_id_constraints = set()
-        exclude_neuron_id_constraint = set()
-        exclude_annotation_map = set()
-        exclude_annotation_ids = list()
+        exclude_skeleton_id_constraints = set() # type: Set
+        exclude_neuron_id_constraint = set() # type: Set
+        exclude_annotation_map = dict() # type: Dict
```

Comment:
I am pretty sure it was a mistake that this was a set initially

Reply:
I agree.